nuc.oscon.2017/README.md

```diff
@@ -0,0 +1,57 @@
+# Crate.io @ OSCON 2017 in Austin, TX
+
+This is a demo that was featured on the Crate.io booth at OSCON in May 2017.
+
+## Contents
+
+A container based face tracking application that requires a Raspberry Pi and Raspberry Pi Camera to work; as well as a CrateDB cluster for the data to be inserted into. The demo will use the created Docker container to fetch images from the camera, apply a face-detection algorithm on each frame and store the results in a CrateDB cluster.
+
+## Requirements
+
+As for the hardware, the requirements are pretty simple:
+
+ - Raspberry Pi 3, set up, with Docker installed
+ - Raspberry Pi (NoIR) Camera
+ - YUM-based Linux machine(s) with access to install CrateDB and software dependencies
+
+## How to use
+
+Please note that this is a demo that was created for an event and might not be maintained at the time you find this. It should stand as an interesting example and provide insights into integrating applications with CrateDB, Raspberry Pis, Ansible, Docker, etc. so expect some tinkering 😊
```

Comment:
wrap this line (and other lines in the doc) to 79 chars

"etc" -> "and so on"

drop "so expect some tinkering 😊"